etl/controller/pulsemaintainer.go

```diff
@@ -24,11 +24,10 @@ func (c *Controller) pulseMaintainer(ctx context.Context) {
 		default:
 			time.Sleep(time.Second * time.Duration(c.cfg.PulsePeriod))
 		}
+		c.jetDropRegisterLock.Lock()
```

Comment:
It would be nice to wrap lock and `for range` in anonymous function, and do `defer c.jetDropRegisterLock.Unlock()` right after lock, so we don't forget unlock in the future.